controller/composite/controller.go

```diff
@@ -499,7 +498,7 @@ func (pc *parentController) claimChildren(parent *unstructured.Unstructured) (co
 	childMap := make(common.ChildMap)
 	for _, child := range pc.cc.Spec.ChildResources {
 		// List all objects of the child kind in the parent object's namespace.
-		childClient, err := pc.dynClient.Resource(child.APIVersion, child.Resource, namespace)
+		childClient, err := pc.dynClient.Resource(child.APIVersion, child.Resource)
```

Comment:
Sorry for not noticing this earlier: I think since we no longer set the namespace here, we need to fix `UnstructuredManager` in the `dynamic/controllerref` package to call `Namespace()` when needed (if the child object has a namespace set).

Reply:
@enisoc Yes, they passed. I ran all the tests for every commit now with image containing my changes via `skaffold run` after changing `skaffold.yaml` and `image.yaml` to my docker ID.

Reply:
Noted, I will change `dynamic/controllerref` package accordingly.

Reply:
@enisoc Do we also need to fix `ControllerRevisionManager`? `ControllerRevision` are namespaced objects.

Reply:
That's a good point, I hadn't thought of that. I just checked though and it looks like ControllerRevisionManager is unaffected by your changes because we use a generated client for that rather than the dynamic client.

Reply:
I see. I will push a commit soon. Thanks!